cargo-shuttle/src/lib.rs

```diff
@@ -242,6 +260,7 @@ impl Shuttle {
             .run_cargo_package(args.allow_dirty)
             .context("failed to package cargo project")?;
 
+
```

Comment:
Double new line? `cargo fmt --all` might automatically fix this for you

Reply:
VSCode has let me down. Fixed, thanks! :)